tests/in/object-select.wgsl

```diff
@@ -0,0 +1,7 @@
+[[stage(fragment)]]
```

Comment:
I don't think we should have an individual test file for every feature of the IR. Let's put this into `operations.wgsl`, can we?